platforms/heroku/index.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express')
+const dotenv = require('dotenv')
+dotenv.config()
+
+const { Photon } = require('./prisma/photonjs')
+
+const photon = new Photon()
+
+const app = express()
+const port = process.env.PORT || 3000
+
+app.get('/', async (req, res) => {
+  const createUser = await photon.users.create({
+    data: {
+      email: 'alice@prisma.io',
+      name: 'Alice',
+    },
+  })
+
+  const updateUser = await photon.users.update({
+    where: {
+      id: createUser.id,
+    },
+    data: {
+      email: 'bob@prisma.io',
+      name: 'Bob',
+    },
+  })
+
+  const users = await photon.users.findOne({
+    where: {
+      id: createUser.id,
+    },
+  })
+
+  const deleteManyUsers = await photon.users.deleteMany()
```

Comment:
afaik this should be `.deleteMany({})`

Reply:
types give a hiccup at times, but this is "js" and runtime is happy with this.